_posts/2022-11-02-gsoc2022-final.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "GSoC / Outreachy 2022 wrap-up"
+---
+
+<p>
+<img
+src="{{ site.baseurl }}{{ site.images }}/mdanalysis-gsoc.png"
```

Comment:
Does outreachy have a logo that we can use?

Reply:
This would probably do the trick. 

https://commons.wikimedia.org/wiki/File:Outreachy-logo.svg